src/withWebViewBridge.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import * as uuid from 'uuid';
 
 export type WebViewWithBridgeProps = {
+    onError: (e: any) => void;
```

Comment:
expose listener when unable to post messages to the webview

Reply:
should this be optional?  not sure who all uses this library, but may result in a breaking change